test-crates/duchess-java-tests/tests/ui/interface_class_mismatch_specified.stderr

```diff
@@ -0,0 +1,8 @@
+error: error in class `exceptions.ThrowExceptions`: class `exceptions.ThrowExceptions` should be type `Class`
```

Comment:
two small nits on this error:
1. the span is wrong—it should be on `interface` and not `public`
2. Should we lower case the output so that it's actually correct?

Reply:
1. Yes, but that's where all the Spans are, even for other things. I'd prefer to punt that to a separate issue where we try and have a better place for the spans.
2. Yeah, that's probably better.